app/views/users/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<h1>All Users</h1>
+<div class="container">
+  <div class="row">
+    <div class="col-12">
+      <table class="table">
+        <thead class="thead-dark">
+          <tr>
+          	<th></th>
+            <th><%= t('user.table.username') %> </th>
+            <th><%= t('user.table.role') %> </th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @users.each do |user| %>
+            <% if user == current_user %>
+              <% next %>
+            <% end %>
+          <tr>
+          	<td>
+              <% if user.profile_picture.attached? %>
```

Comment:
Please check the indentations!

Reply:
I don't understand sorry, what is wrong?